gateway/services/notifications.go

```diff
@@ -5,14 +5,27 @@ import (
 	"github.com/HackIllinois/api/gateway/middleware"
 	"github.com/HackIllinois/api/gateway/models"
 
+	"net/http"
+
 	"github.com/arbor-dev/arbor"
 	"github.com/justinas/alice"
-	"net/http"
 )
 
 const NotificationsFormat string = "JSON"
 
 var NotificationsRoutes = arbor.RouteCollection{
+	arbor.Route{
+		"GetAllSubscriptions",
+		"GET",
+		"/notifications/subscriptions",
+		alice.New(middleware.IdentificationMiddleware, middleware.AuthMiddleware([]models.Role{models.UserRole})).ThenFunc(GetAllSubscriptions).ServeHTTP,
+	},
+	arbor.Route{
+		"GetAllRegisteredDevices",
```

Comment:
I think something more like "GetUserRegisteredDevices" might be better, since you're getting the registered devices of a specific user, not all users.